Lake/DSL/Script.lean

```diff
@@ -11,17 +11,17 @@ namespace Lake.DSL
 open Lean Parser Command
 
 syntax scriptDeclSpec :=
-  ident (ppSpace "(" Term.simpleBinder ")")? (declValSimple <|> declValDo)
+  ident (ppSpace "(" ident+ ")")? (declValSimple <|> declValDo)
 
 scoped syntax (name := scriptDecl)
 (docComment)? "script " scriptDeclSpec : command
 
 @[macro scriptDecl]
 def expandScriptDecl : Macro
-| `($[$doc?]? script $id:ident $[($args?)]? do $seq $[$wds?]?) => do
-  let args := args?.getD (← `(Term.hole|_))
-  `($[$doc?]? @[«script»] def $id : ScriptFn := fun $args => do $seq $[$wds?]?)
-| `($[$doc?]? script $id:ident $[($args?)]? := $defn $[$wds?]?) => do
-  let args := args?.getD (← `(Term.hole|_))
-  `($[$doc?]? @[«script»] def $id : ScriptFn := fun $args => $defn $[$wds?]?)
+| `($[$doc?]? script $id:ident $[($args?*)]? do $seq $[$wds?]?) => do
+  let args := args?.getD (α := Array FunBinder) #[← `(Term.hole|_)]
+  `($[$doc?]? @[«script»] def $id : ScriptFn := fun $args* => do $seq $[$wds?]?)
```

Comment:
Btw, is this branch not covered by the next one?

Reply:
Mostly, yes. It just allows you to write `script foo do` rather than `script foo := do` (which I personally like), though that might be a bit too much syntactic sugar.

Reply:
Ahh, I missed that completely. I rewrote it as a recursive macro, what do you think?